Testreglar/252/App/app-2.5.2a.json

```diff
@@ -0,0 +1,189 @@
+			"type": "jaNei",
+			"kilde": [
+				"F101"
+			],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Ja",
+					"utfall": "Trykkhandlingen (down-event) utløser ikke funksjonen"
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er det avgjørende å utføre funksjonen på trykkhandlingen (down-event)?",
+			"ht": "<p>Dette gjelder tilfeller der tidspunkt (timingen) for en aktivering av funksjonen er viktig og forutsetter ned-hendelsen</p>\n<p>Eksempler på situasjoner der det er avgjørende å utføre funksjonen på trykkhandlingen:</p>\n<ul>\n<li>Funksjoner som krever eller måler reaksjon, for eksempel leirdueskyting spill</li>\n<li>Funksjoner som etterligner et tastatur, for eksempel skjermtastatur jf. merknad til suksesskriteriet</li>\n<li>Funksjoner som etterligner et instrument, for eksempel tangenter på et piano </li>\n<li>Andre simuleringer og spill fra det virkelige liv.</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "ikkjeForekomst",
+					"utfall": "Det er avgjørende å utføre funksjonen på trykkhandlingen (down-event)."
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "3.4"
+				}
+			}
+		},
+		{
+			"stegnr": "3.4",
+			"spm": "Opphever slipphandlingen med pekeren (up-event) resultatet av den foregående trykkhandlingen (down-event)?",
+			"ht": "<p>Sjekk om slipphandlingen med pekeren reverserer eller avbryter resultatet av den foregående trykkhandlingen.</p>\n<ul>\n<li>Trykk og hold nede pekeren du bruker (museknappen, fingeren eller stylus) på innholdet.</li>\n<li>Beveg deretter pekeren vekk fra innholdet, mens du fremdeles holder pekeren nede.</li>\n<li>Slipp opp pekeren og sjekk at dette ikke utfører en handling.</li>\n</ul>\n<p>Eksempler:</p>\n<ul>\n<li>Trykk og hold for å zoome inn: Trykke og holde pekeren på et bilde for å forstørre det.  Når pekeren slippes vises bildet i vanlig størrelse</li>\n<li>Dra og slipp: Trykkhandlingen plukker opp et objekt som følger pekeren, for eksempel et Kanban-brett i Jira.\n<ul>\n<li>Dersom pekeren slippes utenfor slippsonen, går objektet tilbake til utgangspunktet</li>\n<li>Dersom pekeren slippes innenfor slippsonen kommer det opp en dialogboks for å bekrefte eller avbryte</li>\n</ul>\n</li>\n</ul>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.5"
+				},
+				"nei": {
+					"type": "avslutt",
+					"fasit": "Nei",
+					"utfall": "Slipphandlingen (up-event) opphever ikke resultatet av den foregående trykkhandlingen (down-event)"
```

Comment:
På nei skal du vell sjekke mekanisme ref punkt 2 i krav til samsvar: _Funksjonen utføres på slipphandlingen (up-event), og det finnes en mekanisme
- for å avbryte funksjonen før den utføres, eller
- for å angre funksjonen etter at den er utført eller_